app/Notifications/CFPsAreOpen.php

```diff
@@ -2,28 +2,28 @@
 
 namespace App\Notifications;
 
-use App\Conference;
 use Illuminate\Bus\Queueable;
+use Illuminate\Support\Collection;
 use Illuminate\Notifications\Notification;
 use Illuminate\Notifications\Messages\MailMessage;
 
-class CFPIsOpen extends Notification
+class CFPsAreOpen extends Notification
 {
     use Queueable;
 
     /**
      * @var Conference
      */
-    public $conference;
+    public $conferences;
 
     /**
      * Create a new notification instance.
      *
      * @param Conference $conference
```

Comment:
Either fix or delete this docblock please

Reply:
I just removed it since it was really telling nothing more than the property name told anyway.